exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
```

Comment:
In here you can try and tidy up you code a little bit by doing something like this:
`function containsCode(sentence) {return sentence.includes('code');}`
All your work is good, I'm just trying to find something to say. :)